cypress/e2e/build-your-plate.cy.ts

```diff
@@ -1,6 +1,257 @@
+import terminalLog from "../support/terminal-log";
+import foodList from "../fixtures/build-your-plate.json";
+
 describe("Build Your Plate", () => {
-  it("gives correct score with 5 sweets");
-  it("gives correct score with 5 meats");
-  it("gives correct score with 5 mixed items");
-  it("allows user to swap food items before scoring");
+  beforeEach(() => {
+    cy.visit("http://localhost:3000/Games/BuildYourPlate");
+  });
+
+  it("check a11y buuld-your-plate page", () => {
+    cy.injectAxe();
+    cy.checkA11y(null, null, terminalLog, true);
+  });
+
+  it("can navigate to the game page", () => {
+    cy.visit("http://localhost:3000/");
+    cy.get(".react-multi-carousel-list [data-testid='Games']").click();
+    cy.get("[data-testid='menu-styling'] [data-testid='Build your plate']").click();
+    cy.url().should("eq", "http://localhost:3000/Games/BuildYourPlate");
+  });
+
+  it("gives correct score with 5 sweets", () => {
+    cy.get("[data-testid='game-begin']").click();
+    cy.contains("button", "Play").click();
+    cy.contains("button", "Score my plate").should("be.disabled");
```

Comment:
The Score my plate button being disabled prior to any selections could be its own short test.